fl4health/privacy_mechanisms/dpDiscreteGaussian.py

```diff
@@ -0,0 +1,132 @@
+import math
+from typing import List
+
+import torch
+
+
+def Bernoulli_exp(gamma: float) -> int:
+    """
+    Draws a sample from Bernoulli(exp(-gamma))
+
+    Args:
+        gamma (float): A non-negative parameter of type float.
```

Comment:
No need to specify the type in the comment, as it is already annotated in the args typing. An intuitive description of what gamma is would be useful (if there is one)